### ISSUE TITLE
-3^2

### ISSUE THREAD

**Issue by calculatorgr** [bugwheels94/math-expression-evaluator#87]
-3^2 = 9 Should evaluate to -9

note: (-3)^2 should evaluate to 9

**Comment by bugwheels94** [comment]
I will look at the issues over this weekend since all last three issues on repo are same

**Comment by calculatorgr** [comment]
> I will look at the issues over this weekend since all last three issues on repo are similar

You are right. The two operators "^" and "^-" should have higher priority than +, -, *, /, ln, log, sin, cos, tan, asin, acos, ...

**Comment by calculatorgr** [comment]
I prefer to use version 2.0.2 
This version calculates both expressions -3^2 and 1-3^-1 correctly.
The newer versions don't.

**Comment by HATyCEHKA** [comment]
works correct in 2.0.6
